<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Lukáš Tinkl &lt;lukas@redhat.com&gt;</author>
 <class>GsmWidget</class>
 <widget class="QWidget" name="GsmWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>461</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="basicGroup">
     <property name="title">
      <string>Basic</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Number:</string>
        </property>
        <property name="buddy">
         <cstring>number</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="KLineEdit" name="number">
        <property name="text">
         <string notr="true">*99#</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Username:</string>
        </property>
        <property name="buddy">
         <cstring>username</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="KLineEdit" name="username"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Password:</string>
        </property>
        <property name="buddy">
         <cstring>password</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="KLineEdit" name="password">
          <property name="passwordMode">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KComboBox" name="passwordStorage">
          <item>
           <property name="text">
            <string>Store</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Always Ask</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Not Required</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="advGroup">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Advanced</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="3" column="1" colspan="2">
       <widget class="QCheckBox" name="roaming">
        <property name="text">
         <string>Allow roaming if home network is not available</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>APN:</string>
        </property>
        <property name="buddy">
         <cstring>apn</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelNetworkId">
        <property name="text">
         <string>Network ID:</string>
        </property>
        <property name="buddy">
         <cstring>networkId</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="KLineEdit" name="apn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Type:</string>
        </property>
        <property name="buddy">
         <cstring>type</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>PIN:</string>
        </property>
        <property name="buddy">
         <cstring>pin</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="KLineEdit" name="networkId"/>
      </item>
      <item row="2" column="1">
       <widget class="KComboBox" name="type">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="KLineEdit" name="pin">
          <property name="passwordMode">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="KComboBox" name="pinStorage">
          <item>
           <property name="text">
            <string>Store</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Always Ask</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Not Required</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>130</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QCheckBox" name="cbShowPasswords">
     <property name="text">
      <string>Show passwords</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>number</tabstop>
  <tabstop>username</tabstop>
  <tabstop>password</tabstop>
  <tabstop>passwordStorage</tabstop>
  <tabstop>apn</tabstop>
  <tabstop>networkId</tabstop>
  <tabstop>type</tabstop>
  <tabstop>roaming</tabstop>
  <tabstop>pin</tabstop>
  <tabstop>pinStorage</tabstop>
  <tabstop>cbShowPasswords</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
